tests/storage/migration.go

```diff
@@ -587,6 +591,132 @@ var _ = SIGDescribe("Volumes update with migration", decorators.RequiresTwoSched
 				return vm.Status.VolumeUpdateState.VolumeMigrationState
 			}).WithTimeout(120 * time.Second).WithPolling(time.Second).Should(BeNil())
 		})
+
+		Context("should be able to recover from an interuppted volume migration", func() {
+			const volName = "volume0"
+
+			createMigpolicyWithLimitedBandwidth := func(vmi *virtv1.VirtualMachineInstance) {
```

Comment:
Since it's using a policy for a specific VMI I assume it can be parallelized when we graduate the feature gate?

Reply:
Yes, we use the same mechanism also in other tests (cpu hotplug for example). Right now, this is serial because it modifies KubeVirt CR. Although, I think we could enable the feature gate by default and parallelize these tests, but, well, that's for another PR

Reply:
Yeah I just wanted to know it's ready policy-wise